src/App.js

```diff
@@ -1,29 +1,11 @@
-import logo from './logo.svg';
-import './App.css';
+import React from 'react';
+import Calculator from './components/Calculator';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          {' '}
-          <code>src/App.js</code>
-          {' '}
-          and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+// eslint-disable-next-line react/prefer-stateless-function
```

Comment:
- [ ] Please remove the **eslint disable** statement. Because ` Eslint` helps developers to write clean JavaScript codes and adapt best practices when writing the code.  You just need to export it as the default code. :+1:

Reply:
Thanks for the feedback but I'm a little confused on how to make it a default code because everyone seems to disable the Eslint due to the error it shows. Please view the error in my terminal area in the screenshot provided below. Thank you

![image](https://user-images.githubusercontent.com/89091822/183765991-a35c4d47-8e5a-4847-8d1d-c53ff3da5298.png)